specifyweb/frontend/js_src/lib/components/Forms/DeleteButton.tsx

```diff
@@ -202,22 +225,35 @@ export async function fetchBlockers(
               }))
             : await runQuery<readonly [number, number]>(
                 serializeResource(
-                  createQuery('Delete blockers', directRelationship.model).set(
-                    'fields',
-                    [
-                      QueryFieldSpec.fromPath(directRelationship.model.name, [
-                        directRelationship.model.idField.name,
+                  /*
+                   * TODO: Check if this is possible.
+                   */
+                  parentRelationship.otherSideName === undefined
```

Comment:
Check for the possibility if `otherSideName` can be undefined?  

This might be a good place to start this check:

> However, this may happen to other tables and relationships. Using the following regular expression (`<relationship(?!.*othersidename).*`), I was able to find 524 occurrences where relationships do not have the `othersidename` attribute

https://github.com/specify/specify7/issues/2305#issuecomment-1594922611

`otherSideName` is not defined in cases where it does not exist in the xml. 

https://github.com/specify/specify7/blob/312139ad3ee9290dd00500e764dd0f0c36f8b026/specifyweb/specify/load_datamodel.py#L214-L215

Reply:
the case where parent relationship is defined, but no other side name is what we are concerned about here.